### PR TITLE
Chore/cleanup comments files

### PR DESCRIPTION
**Pull request by ItsWendell**

### CODE REVIEW

**Review by Floris497 — COMMENTED** [review]
Looks good on a quickst scroll

File: srcs/vm/args.c [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+**	IS_REGISTRY
+**	Checks if given argument is a valid registry
+*/
+
+int			is_registry(int arg)
+{
+	if (arg > 0 && arg <= REG_NUMBER)
+		return (true);
+	return (false);
+}
```
Comment (JoorsB) [inline srcs/vm/args.c]: .

File: srcs/vm/champions.c [hunk 2]
```diff
@@ -24,4 +24,25 @@ t_champion	*champion_find_id(t_arena *arena, int id)
...
+	int		i;
+
+	i = 0;
+	while (i < MAX_PLAYERS)
+	{
+		if (arena->champions[i].argv_index > 0)
+			arena->champions[i].file_name = argv[arena->champions[i].argv_index];
+		i++;
+	}
 }
```
Comment (JoorsB) [inline srcs/vm/champions.c]: .

**Review by JoorsB — APPROVED** [review]

**Comment by ItsWendell** [comment]
okay yolo merge

**Pull request merged**